README.md

```diff
@@ -53,6 +53,9 @@ OPTIONS:
     -t, --threshold <THRESHOLD>
             Sets the maximum amount of space to be used for Docker images (default: 10 GB)
 
+    -m, --min-age <DURATION>
```

Comment:
Oh sorry, this was my bad, but I think it should be `<MIN AGE>` rather than `<DURATION>`? If I'm reading the code correctly... (can you do `cargo run -- --help` to confirm?)